### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by agherzan**

### CODE REVIEW

File: modules/util.py [hunk 1]
```diff
@@ -531,13 +531,9 @@ def safeDirCopy(src, dst, sync=True, ignore=[]):
 
     # Copy each file in the structure of src to dst
     for root, dirs, files in os.walk(src):
-
             # Directories
+            dirs[:] = list(filter(lambda d: not d in ignore, dirs))
```
Comment (petrosagg) [inline modules/util.py]: NIT: this can also be a comprehension: `dirs[:] = [d for d in dirs if not d in ignore]`

**Comment by floion** [comment]
"util.py: Fix bug when altering the varible on which the iteration is …
…done" : can you describe such a case when the d variable is changes, I assume?

**Comment by floion** [comment]
If it's related to the fact that you should not modify the content of a list while looping over it, then please change the commit log and the subsequent CHANGELOG entry to reflect that fact, rather than just saying this fixes the issues when there are more than one directories to be ignored, which is not the real issue.

**Comment by floion** [comment]
Okay, otherwise (besides a more verbose commit log as noted in my previous comments) approved

File: modules/util.py (hunk 1 above)
Comment (agherzan) [inline modules/util.py]: Addressed.

**Comment by agherzan** [comment]
@floion addressed. Thank you.

**Pull request merged**